src/Trakx.Kaiko.ApiClient/MarketData/AggregatesClient.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Text;
+
+namespace Trakx.Kaiko.ApiClient;
+
+// this is only here to stop Codacy from flagging issue:
+// 'partial' is gratuitous in this context.
+internal partial class AggregatesClient { }
```

Comment:
Yes, really. It seems Codacy analyses files one by one, so it reports partial classes as 'gratuitous' incorrectly.